node/node.go

```diff
@@ -179,6 +179,17 @@ func (n *Node) GetUsername(address string, options api.AccountQueryOptions) (str
 	return string(username), blockInfo, nil
 }
 
+// GetCodeHash gets the code hash for a specific address
+func (n *Node) GetCodeHash(address string, options api.AccountQueryOptions) (string, api.BlockInfo, error) {
+	userAccount, blockInfo, err := n.loadUserAccountHandlerByAddress(address, options)
+	if err != nil {
+		return "", api.BlockInfo{}, err
+	}
+
+	codeHash := userAccount.GetCodeHash()
+	return string(codeHash), blockInfo, nil
```

Comment:
this isn't right. Please have a look over `nodeFacade.GetAccount()` function. 
You will see this code block:
```
        codeHash := accountResponse.CodeHash
	code, _ := nf.node.GetCode(codeHash, options)
	accountResponse.Code = hex.EncodeToString(code)
	return accountResponse, blockInfo, nil
```
Therefore, I suggest that you refactor your code, by calling `n.GetCode` inside this function (`n.GetCodeHash`)

In the end, the code hash should be hexified

Reply:
As @bogdan-rosianu suggested, you should not return the code hash as a string as it might contain special characters that will not be correctly handled by the webserver. So, you should use:
```GO
return hex.EncodeToString(codeHash), blockInfo, nil
```